test/test-network/systemd-networkd-tests.py

```diff
@@ -6328,6 +6329,17 @@ def test_ipv6_route_prefix(self):
         output = check_output(*networkctl_cmd, '--json=short', 'status', env=env)
         check_json(output)
 
+        output = check_output(*networkctl_cmd, '--json=short', 'status', 'veth-peer', env=env)
+        check_json(output)
+
+        # PREF64 or NAT64
+        j = json.loads(output)
+        map = j['NDisc']['PREF64']
+
+        self.assertEqual('64:ff9b::', socket.inet_ntop(socket.AF_INET6, bytearray(map['Prefix'])))
+        self.assertEqual('96', map['PrefixLength'])
+        self.assertEqual('68086310925', map['LifetimeUSec'])
```

Comment:
As far as I understand `LifetimeUSec` can't be tested like this because it's calculated when RAs with PREF64s are received. I think it could be dropped for now and maybe reintroduced eventually to make sure that PREF64s expire. Currently they never expire but that's another issue and has nothing to do with this test.